cfme/tests/networks/nuage/test_events_are_triggered.py

```diff
@@ -71,16 +71,16 @@ def test_creating_entities_triggers_targeted_refresh(targeted_refresh, with_nuag
     """
     sandbox = with_nuage_sandbox
     with targeted_refresh.timeout():
-        targeted_refresh.register_target(sandbox['enterprise'].id, 'Enterprise')
-        targeted_refresh.register_target(sandbox['domain'].id, 'Domain')
-        targeted_refresh.register_target(sandbox['subnet'].id, 'Subnet')
-        targeted_refresh.register_target(sandbox['cont_vport'].id, 'Container vPort (L3)')
-        targeted_refresh.register_target(sandbox['vm_vport'].id, 'VM vPort (L3)')
-        targeted_refresh.register_target(sandbox['l2_domain'].id, 'Domain (L2)')
-        targeted_refresh.register_target(sandbox['l2_cont_vport'].id, 'Container vPort (L2)')
-        targeted_refresh.register_target(sandbox['l2_vm_vport'].id, 'VM vPort (L2)')
-        targeted_refresh.register_target(sandbox['group'].id, 'Security group (L3)')
-        targeted_refresh.register_target(sandbox['l2_group'].id, 'Security group (L2)')
+        targeted_refresh.register_target(sandbox.enterprise['obj'].id, 'Enterprise')
+        targeted_refresh.register_target(sandbox.domain['obj'].id, 'Domain')
+        targeted_refresh.register_target(sandbox.subnet['obj'].id, 'Subnet')
+        targeted_refresh.register_target(sandbox.cont_vport['obj'].id, 'Container vPort (L3)')
+        targeted_refresh.register_target(sandbox.vm_vport['obj'].id, 'VM vPort (L3)')
+        targeted_refresh.register_target(sandbox.l2_domain['obj'].id, 'Domain (L2)')
+        targeted_refresh.register_target(sandbox.l2_cont_vport['obj'].id, 'Container vPort (L2)')
+        targeted_refresh.register_target(sandbox.l2_vm_vport['obj'].id, 'VM vPort (L2)')
+        targeted_refresh.register_target(sandbox.group['obj'].id, 'Security group (L3)')
+        targeted_refresh.register_target(sandbox.l2_group['obj'].id, 'Security group (L2)')
```

Comment:
Honestly, I'd prefer if test would do just 

```
sandbox.l2_group['name']
```

Because test reads a bit strange with this 'obj' thing. I don't mind having 'obj' stored, but since 99% of our tests just accesses 'name' and 'id' I'd add those two keys to the dictionary. Wdyt?

PS: Yes, fixture would be slightly redundant then, but the key aspect IMAO is to have easily _usable_ fixture, not nicest implementation :)